### PR TITLE
hotfix

### PR DESCRIPTION
**Pull request by george43g**
update sample.env

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -5,15 +5,24 @@
...
 TWILIO_ACCOUNT_SID=""
 TWILIO_API_KEY_SECRET=""
 TWILIO_API_KEY_SID=""
+
+## Service accounts
+FIREBASE_CI_SERVICE_ACCOUNT=""
+GOOGLE_APPLICATION_CREDENTIALS=""
+
+## Set custom build and deploy config for ng/nx
+# NG_CONFIG=""
```
Comment (phpgeekfr) [inline sample.env:24]: what is default ?

File: sample.env [hunk 2]
```diff
@@ -5,15 +5,24 @@
...
+# PROJECT_ID="blockframes-yourName"
```
Comment (phpgeekfr) [inline sample.env:25]: same

**Review by phpgeekfr — APPROVED** [review]

File: sample.env (hunk 1 above)
Comment (george43g) [reply sample.env:24]: there is no default. default is blank

File: sample.env (hunk 2 above)
Comment (george43g) [reply sample.env:25]: no that's just an example. again, no default. whatever firebase is currently `use` isThe default

File: sample.env (hunk 1 above)
Comment (phpgeekfr) [reply sample.env:24]: Can you give an example value ? And why it should be used ?

File: sample.env (hunk 2 above)
Comment (phpgeekfr) [reply sample.env:25]: OK this is to override `firebase use` . Can you explain this in comment and explain in which context it should be used?

Add a link to notion page if there is an existing one for this

File: sample.env (hunk 1 above)
Comment (george43g) [reply sample.env:24]: example value:
`production` to compile for production 
`staging` used by demo environments


... come to think of it, the `PROJECT_ID` could be extracted from the config, as each one corresponds to a projectId , for example - 

NG_CONFIG=production
PROJECT_ID=blockframes

Comment (george43g) [reply sample.env:24]: these are mainly used by CI, but if a user wanted to work on demo1, staging, for example, they could set these and run nx deploy XXX

but they can also run nx deploy xxx --configuration=demo1 if they want to. atm both options work, mainly used by us

**Pull request merged**